utils/details.js

```diff
@@ -115,8 +119,33 @@ class ActionDetails {
           if (line.indexOf('---') === -1 && line.indexOf('+  - ') === 0) {
             const slice = line.slice(1).trim()
 
-            // eslint-disable-next-line no-unused-vars
-            const [_, o, r, v] = actionRegEx.exec(slice)
+            if (slice.indexOf('docker://') > -1) {
+              const match = dockerRegEx.exec(slice)
+
+              if (!match) {
+                actions.push({
+                  owner: 'docker://',
+                  repo: null,
+                  version: null,
+                  position: i,
+                  registry: slice.slice(11),
```

Comment:
Is this intended to match everything after `docker://`?  If so, shouldn't the starting position be sooner especially since `slice` effectively trims off `+`/`-`?

```
                  registry: slice.slice(9),
```

Reply:
The string is actually `- docker://...` so `slice.slice(11)` is needed.